pyactiveresource/util.py

```diff
@@ -196,7 +196,9 @@ def camelize(word):
         The modified string.
     """
     return ''.join(w[0].upper() + w[1:]
-                   for w in re.sub('[^A-Z^a-z^0-9^:]+', ' ', word).split(' '))
+        for w
+        in re.sub('[^A-Z^a-z^0-9^:]+', ' ', word.strip()
+        ).split(' '))
```

Comment:
Hahaha. Nope. It's really not much easier to read but it was 107 cols wide when not split onto many lines. It felt like a bit much.

    return ''.join(w[0].upper() + w[1:] for w in re.sub('[^A-Z^a-z^0-9^:]+', ' ', word.strip()).split(' '))¬

I don't mind making it all one line if you would prefer? @kevinhughes27